### DISCUSSION TITLE
Point2grid - grid specification string

### DISCUSSION THREAD

**Discussion by spence98** [dtcenter/METplus#1481] [Configuration] [answered]
Hello,

I am trying to grid my LSRs to a 3km grid, following the use case example, but using a grid specification string based off the wrf data I will use for verification purposes. I am getting an error when doing so, but am not sure how to address it. I'm attaching my configure file and the output log file that was created when I tried to run it. 

Thanks for your help!

P.S. I have successfully processed the LSRs using that configuration file for the standard ~80x80km grid, so I know that it was working prior to changing the grid to the grid specification string.

[Archive.zip](https://github.com/dtcenter/METplus/files/8188921/Archive.zip)

**Comment by JohnHalleyGotway** [comment, marked as answer]
@spence98 thanks for sending the sample data to demonstrate the issue. Checking the METplus log file you sent, I see that METplus is trying to execute this command:
```
/usr/local/bin/point2grid /metplus/output/lsr_data/13oct14_StormReps_bi_3.nc lambert 1200 1200 17.60836 -103.5123 -87.5 3.0 6367.47 30.0 N /metplus/output/lsr_data/13oct14_StormReps_3_bi.nc -field 'name="Fscale"; level="";' -method MAX -gaussian_dx 0 -gaussian_radius 0 -v 10
```

Checking the formatting of your grid spec string against the documentation in this [appendix](https://met.readthedocs.io/en/develop/Users_Guide/appendixB.html?highlight=lambert#grid-specification-strings), I don't see any issues:
```
lambert Nx Ny lat_ll lon_ll lon_orient D_km R_km standard_lat_1 [ standard_lat_2 ] N|S
```

I think this is just a pesky little issue with quotes. Point2Grid processes the "to_grid" as a single option. When defining a grid spec string, we need to enclose it in single quotes so that it's parsed as a single option on the command line. Please try updating your METplus conf file using:
```
POINT2GRID_REGRID_TO_GRID = 'lambert 1200 1200 17.60836 -103.5123 -87.5 3.0 6367.47 30.0 N'
```

Does that produce any better results?

**Comment by spence98** [comment]
Hello John,

Thank you for your help in this! Once I added the single quote I was able to successfully process the LSRs onto my defined grid!

Michelle

**Reply by JohnHalleyGotway** [reply]
WooHoo! Thanks for confirming Michelle. I coordinated with @georgemccabe on this to see if there's any logic that could be added to the METplus wrapper to wrangle quotes for us when needed.

**Reply by georgemccabe** [reply]
I am going to update the logic to handle the quotes properly so in 4.1.0 you shouldn't need to add single quotes around the value, although it will not hurt to include them.
